docker/activemq.yml

```diff
@@ -1,12 +1,14 @@
+version: ''
```

Comment:
Please remove the newly added `version` properties. Docker should warn you about it. At my machine it outputs: "`version` is obsolete"

Reply:
It is, but some IDE's (including IntelliJ) still use this to check if it is a Dockerfile. That's why I added it, but kept it empty.

Reply:
I didn't have issues because of that. Are you using the latest IntelliJ version and latest plugin versions?

Reply:
Oh, that's interesting, I'd like to troubleshoot that a bit.

I have version `Build #IU-241.15989.150, built on April 29, 2024` of IntelliJ and `bundled 241.15989.150` for the Docker plugin.

Reply:
That's not the newest, I'll update

Reply:
It wasn't fiexed